frontend/components/panels/SurplusAuctionRestartPanel.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name" class="SurplusAuctionRestartPanel">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+
+        <TextBlock v-if="isExplanationsShown">
+            Auction that was started but hasn't received a single bid in the allotted time. When restarted bids can be
+            placed again.
+        </TextBlock>
```

Comment:
Since this is the only actual difference from the original `AuctionRestartPanel` I think it might be the better idea to merge these into one component and select the respective `TextBlock` with a new prop `auctionType` like we did in #334

Reply:
would pass this question to @valiafetisov since it is code base related...